core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -151,16 +154,31 @@ export class CkEditorInitializerService {
                   },
                   function(widgetShouldBeRemoved) {
                     if (widgetShouldBeRemoved || that.data.isCopied) {
+                      const highlighted = customizationArgSpecs
```

Comment:
The name could be better here. Maybe just copy default_value_obtainable_from_highlight, I dunno. `highlighted` gives impression that it's a flag that some text is highlighted, or it's a highlighted text

Reply:
Done